Tests/Resources/app/config/config.php

```diff
@@ -10,6 +10,7 @@
  */
 
 $container->setParameter('cmf_testing.bundle_fqn', 'Symfony\Cmf\Bundle\ResourceRestBundle');
+$container->setParameter('kernel.environment', 'test');
```

Comment:
should not be needed if the Kernel is instantiated correctly (with the test env).